cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -82,7 +81,10 @@ def test_vm_mac_scvmm():
             1.
             2. MAC Address should match what is in SCVMM
     """
-    pass
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.all()[0]
+    # compare mac_address in CFME to that from SCVMM
+    assert vm.mac_address == vm.mgmt.raw["VirtualNetworkAdapters"][0]["PhysicalAddress"]
```

Comment:
I don't like these `[0]`. If there are more vms, the test can start false-failing.

Reply:
@jaryn Why would this fail with more vms? It will just take the first one. Every provider has at least `cu-24x7` on it. For the `VirtualNetworkAdapters`, most VMs have only one of those, but the API returns a list, so we take the first one.

Reply:
@john-dupuy what makes you think api returns objects in same order as web-ui?

Reply:
@jaryn Good point, I can modify this to get a list of mac addresses and ensure that the one in the web-ui is in that list.